lighthouse-cli/test/fixtures/dobetterweb/dbw_tester.html

```diff
@@ -306,8 +306,6 @@ <h2>Do better web tester page</h2>
     MediaStreamTrack.getSources(); // Should eventually FAIL
   } catch(e) {}
 
-  console.timeline(); // Should eventually FAIL
```

Comment:
the `Should eventually FAIL` seems to be a signal that at some point in the future we expected that this would trigger a deprecation? 

we assert that 3 deprecations are found in our expectations, the 3 real ones + 2 fake ones (1 fake one) after this PR